packages/apify/src/configuration.ts

```diff
@@ -22,6 +22,7 @@ export interface ConfigurationOptions extends CoreConfigurationOptions {
     proxyPort?: number;
     proxyStatusUrl?: string;
     standbyPort?: number;
```

Comment:
Is it possible to mark this property as deprecated?

Reply:
done, added.